test/colony-funding.js

```diff
@@ -634,43 +689,68 @@ contract("Colony Funding", accounts => {
-      await colony.startNextRewardPayout(otherToken.address);
+      await colony.startNextRewardPayout(otherToken.address, ...colonyWideReputationProof);
 
-      await checkErrorRevert(colony.startNextRewardPayout(otherToken.address), "colony-reward-payout-token-active");
+      await checkErrorRevert(colony.startNextRewardPayout(otherToken.address, ...colonyWideReputationProof), "colony-reward-payout-token-active");
     });
 
     it("should be able to collect rewards from multiple payouts of different token", async () => {
       const tokenArgs = getTokenArgs();
       const newToken = await Token.new(...tokenArgs);
       await fundColonyWithTokens(colony, newToken, initialFunding.toString());
 
-      const tx1 = await colony.startNextRewardPayout(newToken.address);
+      const tx1 = await colony.startNextRewardPayout(newToken.address, ...colonyWideReputationProof);
       const payoutId1 = tx1.logs[0].args.id;
 
-      const tx2 = await colony.startNextRewardPayout(otherToken.address);
+      const tx2 = await colony.startNextRewardPayout(otherToken.address, ...colonyWideReputationProof);
       const payoutId2 = tx2.logs[0].args.id;
 
-      await colony.claimRewardPayout(payoutId1, initialSquareRoots, userReputation.toString(), totalReputation.toString(), {
+      await colony.claimRewardPayout(payoutId1, initialSquareRoots, ...userReputationProof1, {
         from: userAddress1
       });
 
-      await colony.claimRewardPayout(payoutId2, initialSquareRoots, userReputation.toString(), totalReputation.toString(), {
+      await colony.claimRewardPayout(payoutId2, initialSquareRoots, ...userReputationProof1, {
         from: userAddress1
       });
     });
 
     it("should not be able to claim payout if colony-wide reputation is 0", async () => {
-      const { logs } = await colony.startNextRewardPayout(otherToken.address);
-      const payoutId = logs[0].args.id;
+      const tokenArgs = getTokenArgs();
+      const newToken = await Token.new(...tokenArgs);
+      const { logs } = await colonyNetwork.createColony(newToken.address);
+      const { colonyAddress } = logs[0].args;
+      const newColony = await IColony.at(colonyAddress);
+      await newToken.mint(10);
+      await newToken.transfer(userAddress1, 10);
+
+      const result = await colony.getDomain(1);
+      const rootDomainSkill = result.skillId;
+      const colonyWideReputationKey = makeReputationKey(colony.address, rootDomainSkill.toNumber());
+      const { key, value, branchMask, siblings } = await miningClient.getReputationProofObject(colonyWideReputationKey);
+      colonyWideReputationProof = [key, value, branchMask, siblings];
 
       await checkErrorRevert(
-        colony.claimRewardPayout(payoutId.toString(), initialSquareRoots, 0, 0, {
-          from: userAddress1
-        }),
+        newColony.startNextRewardPayout(otherToken.address, ...colonyWideReputationProof),
         "colony-reward-payout-invalid-total-reputation"
```

Comment:
You have removed this require string from the contracts, so this test cannot possibly be testing for it. Is it meant to be `colony-reward-payout-invalid-colony-wide-reputation`? If so, this test is wrong, because according to coverage, that require is never tripped.